runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -27,26 +27,27 @@
 #include <errno.h>
 #endif /* JAVA_SPEC_VERSION >= 20 */
 
+#include "bcnames.h"
+#include "ffi.h"
+#include "j2sever.h"
 #include "j9.h"
+#include "j9bcvnls.h"
 #include "j9cfg.h"
-#include "j9protos.h"
 #include "j9consts.h"
-#include "j9vmnls.h"
 #include "j9jclnls.h"
-#include "j9bcvnls.h"
-#include "bcnames.h"
+#include "j9protos.h"
+#include "j9vmnls.h"
+#include "jitregmap.h"
+#include "jni.h"
+#include "jvminit.h"
+#include "objhelp.h"
 #include "rommeth.h"
 #include "stackwalk.h"
 #include "ut_j9vm.h"
 #include "util_api.h"
 #include "vm_internal.h"
-#include "jni.h"
-#define FFI_BUILDING /* Needed on Windows to link libffi statically */
-#include "ffi.h"
-#include "jitregmap.h"
-#include "j2sever.h"
 #include "vmaccess.h"
-#include "objhelp.h"
+#define FFI_BUILDING /* Needed on Windows to link libffi statically */
```

Comment:
Seems this needs to be defined before ffi.h is loaded.

Reply:
Fixed